web/gridt_server/tests/unit/test_schemas.py

```diff
@@ -1,152 +0,0 @@
-from marshmallow import ValidationError
-
-from gridt_server.tests.base_test import BaseTest
-from gridt_server.schemas import (
-    MovementSchema,
-    NewUserSchema,
-    RequestEmailChangeSchema,
-    ChangeEmailSchema,
-)
-
-from freezegun import freeze_time
-from gridt_server.models.user import User
-
-
-class SchemasTest(BaseTest):
-    def test_movement_schema_long(self):
-        with self.app_context():
-            proper_movement = {
-                "name": "flossing",
-                "short_description": "Flossing everyday keeps the dentist away.",
-                "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus eget sapien ipsum. Nulla eget felis id mi maximus vestibulum a ac lorem. Ut eget arcu sed urna pellentesque hendrerit eu eget ipsum. Sed congue scelerisque dapibus. Suspendisse neque mi, vehicula vel malesuada at, pretium non sapien. Phasellus mi ex, congue.",
-                "interval": "daily",
-            }
-
-            # Make sure no error is thrown with this info
-            schema = MovementSchema()
-            res = schema.load(proper_movement)
-            self.assertEqual(res, proper_movement)
-
-    def test_movement_schema_short(self):
-        with self.app_context():
-            proper_movement = {
-                "name": "flossing",
-                "short_description": "Flossing sometimes is good for you.",
-                "interval": "weekly",
-            }
-
-            # Make sure no error is thrown with this info
-            schema = MovementSchema()
-            res = schema.load(proper_movement)
-            self.assertEqual(res["name"], "flossing")
-
-    def test_movement_schema_bad_interval(self):
-        with self.app_context():
-            bad_movement = {"name": "flossing", "interval": "daily"}
-
-            schema = MovementSchema()
-            with self.assertRaises(ValidationError) as error:
-                schema.load(bad_movement)
-
-                self.assertEqual(
-                    error.exception.messages,
-                    {"short_description": ["Missing data for required field."],},
-                )
-
-    def test_movement_schema_lengths(self):
-        bad_movement = {
-            "name": "flo",
-            "short_description": "1234567",
-            "interval": "daily",
-        }
-
-        schema = MovementSchema()
-        with self.assertRaises(ValidationError) as error:
-            schema.load(bad_movement)
-
-            self.assertEqual(
-                error.exception.messages,
-                {
-                    "short_description": ["Length must be between 10 and 100."],
-                    "name": ["Length must be between 4 and 50."],
-                },
-            )
-
-    def test_request_email_change_schema_invalid(self):
-        with self.app_context():
-            user = self.create_user()
```

Comment:
This test does use models.

Reply:
Forgive my sloppy review. Here, I mean that removing this is fine. In SchemasTest, you should probably restore the tests that do not rely on models.